packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/checkfortoken.md

```diff
@@ -1,36 +1,42 @@
-Check for an existing session in two steps.
+To check for an existing session may take two steps:
 
-1. Initialize the authorization state manager (`authStateManager`) from a saved session.
-2. Verify that the access token is still valid by checking the expiry.
+- Check for an existing valid token.
+- Refresh the token if it has expired.
 
-The state manager session and its associated token are already saved by the `signIn` function calling `writeToSecureStorage`.
+The `Credential` class manages the tokens for a user. The `signIn()` function you wrote in [Open the sign-in page](#open-the-sign-in-page) calls `store(_:)` to save the users' credentials. A session exists if there's a default credential. Although there are calls to check if a credential has expired and to request a refresh, this code uses `refreshIfNeeded()` that only tries to refresh the token if it's expired.
```

Comment:
Why have you started adding `()` to the function names in the text here? There are many references to `signIn`, `signOut`, `showError` etc earlier on that don't have the signature part.

Reply:
This may be partially a revision issue and also one to do with a mix of code the developer writes versus code in the SDK. I'm not sure which direction to go on this and would love to hear what you think. In the above text I use the full signature of the SDK function because it's from the SDK and there are other "store" functions with different signatures.